compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/expressions/DynamicParamNode.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
2021?

Reply:
No, it should be 2020, since the file was created in 2020.

Reply:
Is it when we created or when the PR is getting merged (i.e class is getting added to the repo for the first time)

Reply:
That is not practical. What matters is the commit and when it created.

Reply:
Ack